config/initializers/devise.rb

```diff
@@ -0,0 +1,299 @@
+# frozen_string_literal: true
+
+# Use this hook to configure devise mailer, warden hooks and so forth.
+# Many of these configuration options can be set straight in your model.
+Devise.setup do |config|
+  # The secret key used by Devise. Devise uses this key to generate
+  # random tokens. Changing this key will render invalid all existing
+  # confirmation, reset password and unlock tokens in the database.
+  # Devise will use the `secret_key_base` as its `secret_key`
+  # by default. You can change it below and use your own secret key.
+  # config.secret_key = '666c38a4e620670a693b007038508a3536ff6101142469fbc0311b7ac7c7bc697b01c4ec60e42b0f16ce13d276595c46173817ef6341c4c70197e0af427d9b22'
```

Comment:
Maybe don't commit this? instead put it in a secrets file and use env

Reply:
now dealt with